cli/command/network/connect.go

```diff
@@ -43,6 +47,8 @@ func newConnectCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.Var(&opts.links, "link", "Add link to another container")
 	flags.StringSliceVar(&opts.aliases, "alias", []string{}, "Add network-scoped alias for the container")
 	flags.StringSliceVar(&opts.linklocalips, "link-local-ip", []string{}, "Add a link-local address for the container")
+	flags.Var(&opts.networkOpts, "network-opt", "Set network driver option for a container endpoint")
+	flags.Var(&opts.ipamOpts, "ipam-opt", "Set IPAM driver option for a container endpoint")
```

Comment:
To conform to the help line for the network driver options in the network create command (in network_create.go), please use the plural `options`